packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -325,6 +343,28 @@ class _RenderVectorGraphics extends RenderBox {
       return;
     }
     _pictureInfo = value;
+    invalidateRaster();
+    markNeedsPaint();
+  }
+
+  ColorFilter? get colorFilter => _colorFilter;
+  ColorFilter? _colorFilter;
+  set colorFilter(ColorFilter? value) {
+    if (colorFilter == value) {
+      return;
+    }
+    _colorFilter = value;
+    markNeedsPaint();
```

Comment:
invalidateRaster, no?

Reply:
don't need to re-rasterize, the filter is applied to the drawImage call

Reply:
Ahh ok. I wonder how much of a difference it makes to apply the filter to the image itself. But this is fine.

Reply:
hmm, good point. I'll file a bug for this and we can investigate later